crates/hotshot/src/lib.rs

```diff
@@ -180,7 +202,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> SystemContext<TYPES, I> {
     ) -> Result<Arc<Self>, HotShotError<TYPES>> {
         debug!("Creating a new hotshot");
 
-        let consensus_metrics = Arc::new(metrics);
+        let consensusmetrics = Arc::new(metrics);
```

Comment:
should this be `consensus_metrics`?

Reply:
yeah find and replace of _metrics got me